twelvebit-sensor-data.js

```diff
@@ -0,0 +1,27 @@
+function isValidNumber(number) {
+    const flag = (number < 0 || number > 4094) ? false : true;
```

Comment:
hardcoded values such as 0 and 4094 might lead to error at a later point of time during maintenance.
Are there tests to safeguard assumptions on the hardcoded values?